mipengine/common/resources/pathologies_metadata/dementia.json

```diff
@@ -1531,6 +1531,26 @@
     },
     {
       "variables": [
+        {
+          "isCategorical": true,
+          "code": "alzheimerbroadcategory_bin",
+          "sql_type": "int",
```

Comment:
I am not sure but I think that a categorical variable should not be treated as an "int" type.

An algorithm that can run on numbers should not run on this one because the integer values are enumerations, not actual numbers.

@jassak Please correct me if I am wrong.

Reply:
@ThanKarab This is just some dummy variable we used in the demo, but yes categorical can be `int`. 

In general: `categorical in {int, str}` and `numerical in {int, float}`